.github/workflows/tests_e2e.yml

```
@@ -25,6 +25,12 @@ jobs:
 
       - uses: actions/setup-node@v1
 
+      - name: Install firebase CLI
+        run: npm i -g firebase-tools
+
+      - name: Start Firestore Emulator
+        run: firebase emulators:start  
```

```suggestion
        run: nohup firebase emulators:start  
```

Otherwise CI will sit here waiting for emulator to exit 😅